plugin-template

```diff
@@ -37,6 +37,7 @@ DEFAULT_SETTINGS = {
     'plugin_name': None,
     'plugin_snake': None,
     'plugin_app_label': None,
+    'pulp_settings': None,
```

Comment:
I thought this would get the content_host var?

Reply:
It's up to the plugin author to provide that setting if the plugin needs it.